assets/src/edit-story/components/canvas/displayLayer.js

```diff
@@ -47,33 +118,35 @@ function DisplayLayer() {
     }) => ({ editingElement, setPageContainer, setFullbleedContainer })
   );
 
+  const resetAnimationState = useCallback(
+    () => updateAnimationState({ animationState: STORY_ANIMATION_STATE.RESET }),
+    [updateAnimationState]
+  );
+
   return (
-    <Layer
-      data-testid="DisplayLayer"
-      pointerEvents="none"
-      aria-label={__('Display', 'web-stories')}
+    <StoryAnimation.Provider
+      animations={currentPage?.animations}
+      onWAAPIFinish={resetAnimationState}
```

Comment:
If you also pass in `currentPage?.elements` then it'll be future proofed and ready for all our effects.